src/lib.rs

```diff
@@ -216,6 +216,15 @@ use crate::util::{slice_as_uninit_mut, slice_assume_init_mut};
 use core::mem::MaybeUninit;
 
 mod error;
+// Conditionally import lazy, as not all targets support atomics.
+#[cfg(any(
+    target_os = "android",
+    target_os = "linux",
+    target_os = "netbsd",
+    target_arch = "x86_64",
+    target_arch = "x86"
+))]
+mod lazy;
```

Comment:
I don't like this change. Maybe it's better to use the following code in `rdrand`?
```rust
#[path = "lazy.rs"]
mod lazy;
```

Reply:
I think in general we should probably try to reduce the `#[path]` stuff that's being done. I think we could lust leave `lazy` the way it currently it, or move these changes to a separate PR, since I don't see how they are related to the testing.

Reply:
It's related to the testing changes because of the `rdrand` module. It requires `lazy` and this PR mounts `rdrand` for x86(-64) targets during testing. Without doing this and leaving `rdrand` unchanges we would get `lazy` mounted twice during testing.

Reply:
Lazy changes moved to #472 which then avoids us needing to change any imports for the testing PRs.